Add "Video translation" section in "Microsoft Edge Privacy Whitepaper" (#3239)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

Author: @sidpandey3006 could you please post a Comment here in the Files Changed tab of this PR, such as "ok" in reply to the present comment?  
I'm trying to confirm how GitHub PR UI works: I strongly believe that if you post a comment here, you'll be added to the Reviewers list on main page of PR.  
I cannot add you directly to the Reviewers list in main page of PR - many ppl have tried & failed - seems like a GitHub bug; the list of available ppl for review is wrong.

1. Turn off the **Offer to translate pages that aren't in a language you read** toggle and close the webpage.

<!-- ====================================================================== -->
## Video translation

Video translation provides real-time audio translation of videos you're watching, with audio dubbing in your preferred language.

> The first time you initiate audio translation, a set of language models is downloaded and installed on your device.  Video translation is done locally, on your device.  No data from the video is sent to Microsoft.

Reviewer: ```suggestion
The first time you initiate video translation, a set of language models is downloaded and installed on your device.  Video translation is done locally, on your device.  No data from the video is sent to Microsoft.
```

Author: reworded "audio translation" 2x

1. Turn off the **Offer to translate pages that aren't in a language you read** toggle and close the webpage.

<!-- ====================================================================== -->
## Video translation

Video translation provides real-time audio translation of videos you're watching, with audio dubbing in your preferred language.

> <!-- ------------------------------ -->
> #### Use Video Translate through the video hover menu
> 
> To start playing audio in your preferred language, hover over a video, select the **Translate** icon, and then select the **From** and **To** languages.
> 
> The automatically generated audio might have errors, because the accuracy of the audio translation is affected by factors such as background music, number of speakers, and source language.
> 
> If you want to disable the feature, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages** (or go directly do `edge://settings/languages`), and then turn off the toggle **Offer to translate videos on supported sites**.
> 
> 
> <!-- ------------------------------ -->
> #### Privacy and data protection
> 
> The audio translation is done completely on your device; no data from the video leaves your device, and no segment of the video or audio content ever exits your device or is processed in the cloud.  The first time that initiate audio translation, a set of language models is downloaded and installed on your device.

Reviewer: Removed sections that do not pertain to privacy. Edited the privacy related sections for clarity.

```suggestion
The first time you initiate audio translation, a set of language models is downloaded and installed on your device. Video translation is done locally, on your device. No data from the video is sent to Microsoft. 

To disable video translation, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the **Offer to translate videos on supported sites** toggle.
```

Author: resulting entire h2 section:
```
<!-- ====================================================================== -->
## Video translation

Video translation provides real-time audio translation 
of videos you're watching, with audio dubbing in your 
preferred language.

The first time you initiate audio translation, a set of 
language models is downloaded and installed on your 
device.  Video translation is done locally, on your 
device.  No data from the video is sent to Microsoft.

To disable video translation, select **Settings and more** 
(![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Languages**, and then turn off the **Offer 
to translate videos on supported sites** toggle.
```
@apryzgoda - I re-tested Settings UI steps insofar as possible:
* added my missing step > Settings
* the toggle isn't present yet in Stable or Canary
